### ISSUE TITLE
IconButton: does not follow high contrast standards on Edge

### ISSUE THREAD

**Issue by gaurav42** [OfficeDev/office-ui-fabric-react#5074]
### Bug Report

- __Browser and OS versions__: Windows 10 RS3
- __Microsoft Alias__: gdesh

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority: (Blocking, High, Normal, Low) 
Normal

Products/sites affected: (if applicable)
VS marketplace

#### Describe the issue:
IconButton is not following high contrast color standards in Edge browser

#### Actual behavior:
No change in background and foreground colors of icon button. Only double border comes on focus.

![image](https://user-images.githubusercontent.com/13743203/40908608-e45591b0-6804-11e8-821e-9734267b94a1.png)

![image](https://user-images.githubusercontent.com/13743203/40908641-f7c65c34-6804-11e8-9a95-c966dad1a45d.png)

#### Expected behavior:
In HCW:
Button texts should be in White color with violet background.
In HCB:
Button texts should be in Black color with blue background.

**Comment by michaelangotti** [comment]
Firefox does not support High Contrast modes or does so inconsistently. We only support the high-contrast mode on IE/Edge. Closing this issue.

**Comment by gaurav42** [comment]
@michaelangotti sorry that my issue description said Firefox browser. The issue is also in Edge browser as the title mentions. I have updated the issue description to mention Edge browser.

Can you please reopen the issue or shall I open a new issue?

**Comment by JasonGore** [comment]
What are the high color contrast standards? Are they industry standards or Microsoft standards? Can you provide links in your issue description? Thanks

**Comment by gaurav42** [comment]
In HCW:
Button texts should be in White color with violet background.
In HCB:
Button texts should be in Black color with blue background.

**Comment by JasonGore** [comment]
This is just a repeat of what you had in the description and mentions nothing about icons. However, the problem title isn't written for "button texts", it's written for IconButtons. Don't regular text buttons also exhibit this issue?

Please link to the actual spec or standard so that we can have the additional detail to address correctly. Thanks!

![image](https://user-images.githubusercontent.com/26070760/42177026-edb1b6f8-7ddf-11e8-94ab-b0c52ebed2e6.png)

**Comment by gaurav42** [comment]
The standards were reported to us by accessibility testers. But I checked color palette in Windows 10, which is in sync with the current button colors:

![image](https://user-images.githubusercontent.com/13743203/42199152-7151913e-7eaa-11e8-86ac-d1300bce5abd.png)

![image](https://user-images.githubusercontent.com/13743203/42199156-7c05d1e4-7eaa-11e8-9a61-dd71c9341d7e.png)

I think we can close the issue. I will follow up with the accessibility testers whether there are any additional guidelines.

**Comment by JasonGore** [comment]
Ok, I will close the issue for now. Please feel free to post back any new findings. Thanks
